__tests__/app.test.js

```diff
@@ -305,3 +305,9 @@ describe("GET /api/reviews/:review_id/comments", () => {
       })
   });
 });
+
+describe("GET /api/categories", () => {
+  test("Check it runs", ()=>{
```

Comment:
Please make sure you're following TDD, and writing the tests first, before the functions